shared/src/test/kotlin/com/egm/stellio/shared/model/JsonLdEntityTests.kt

```diff
@@ -0,0 +1,63 @@
+package com.egm.stellio.shared.model
+
+import com.egm.stellio.shared.util.DEFAULT_CONTEXTS
+import com.egm.stellio.shared.util.JsonLdUtils.expandJsonLdEntity
+import com.egm.stellio.shared.util.matchContent
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType
+
+class JsonLdEntityTests {
+    private val mapper: ObjectMapper =
+        jacksonObjectMapper()
+            .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+            .findAndRegisterModules()
+            .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+
+    @Test
+    fun `it should compact and return a json entity`() {
```

Comment:
s/json/JSON

Reply:
You mean the test title should be updated ?

Reply:
yes

Reply:
Name contains illegal characters: /

Reply:
there is no `/`. I suggested to replace json by JSON.

Reply:
(this notation is inspired by the popular `sed` utility)

Reply:
Got it